default.build

```diff
@@ -319,10 +319,21 @@
 			<in>
 				<items>
 					<include name="${nuget.nupackages.dir}/*.nupkg"/>
+					<exclude name="${nuget.nupackages.dir}/*.symbols.nupkg"/>
 				</items>
 			</in>
 			<do>
-				<echo message="nuget push ${filename} ${environment::newline()}" file="${nuget.nupackages.pushbatfile}" append="true"/>
+				<echo message="nuget push -source https://api.nuget.org/v3/index.json ${path::get-file-name(filename)} ${environment::newline()}" file="${nuget.nupackages.pushbatfile}" append="true"/>
```

Comment:
Destination ("source") is become mandatory. [Documentation](https://docs.microsoft.com/en-us/nuget/create-packages/publish-a-package) uses this v3 url, aligning on it.

Adding also `path::get-file-name` because otherwise this was emitting the full path to the package, which is specific to the build machine and will very likely not be right after having downloaded the artifacts from TeamCity.